source/incident-collaboration/launching-playbooks.rst

```diff
@@ -77,29 +75,29 @@ Incident status updates ensure that stakeholders remain informed about the progr
  * Status updates are posted to the incident channel as a message from the user providing the status update.
  * If the playbook has a defined broadcast channel, status updates are copied to the broadcast channel as a message from the incident bot.
 
-The most recent status post will also appear in the right-hand sidebar of the incident channel. To correct or remove a status post, edit or delete the post as needed. Status updates that are broadcast to another channel will not be edited or deleted.
+The most recent status post will also appear in the right-hand sidebar of the playbook channel. To correct or remove a status post, edit or delete the post as needed. Status updates that are broadcast to another channel will not be edited or deleted.
 
-Ending an incident
-~~~~~~~~~~~~~~~~~~
+Ending playbook run
+~~~~~~~~~~~~~~~~~~~
 
-Incident members can end an incident using the ``/incident end`` slash command. Incidents can also be ended from the desktop or browser:
+Playbook run members can end an incident using the ``/playbook end`` slash command. Playbook runs can also be ended from the desktop or browser:
 
-1. Select the **Incidents** icon in the channel header to open the right-hand sidebar.
-2. Find the incident you're looking for and select **Go to Incident Channel**.
+1. Select the **Playbook** icon in the channel header to open the RHS.
+2. Find the incident you're looking for and select **Go to channel**.
 3. Select **Update Status**.
 4. From the **Status** drop-down menu, select **Resolved**.
 5. Enter a message with additional details.
 6. Select **Update Status**.
 
-Ending an incident signals to all members of the channel that the incident is no longer ongoing. Members of the team can continue to post in the channel, mark tasks as complete, and change the commander if needed.
+Ending a playbook run signals to all members of the channel that the playbook run is complete. Members of the team can continue to post in the channel, mark tasks as complete, and change the owner if needed.
 
-Restarting an incident
-~~~~~~~~~~~~~~~~~~~~~~
+Restarting a playbook run
+~~~~~~~~~~~~~~~~~~~~~~~
 
-If an incident was ended prematurely, it can be restarted within the incident channel using the ``/incident restart`` slash command. Incidents can also be restarted from the desktop or browser:
+If playbook run was ended prematurely, it can be restarted within the channel using the ``/playbook restart`` slash command. Runs can also be restarted from the desktop or browser:
```

Comment:
Not sure

```suggestion
If a playbook run was ended prematurely, it can be restarted within the channel using the ``/playbook restart`` slash command. Runs can also be restarted from the desktop or browser:
```